### PR TITLE
[MDK]: updated tutorial with latest changes

### PR DESCRIPTION
**Pull request by jitendrakansal**
updated MDK tutorial with latest changes

### CODE REVIEW

File: tutorials/cp-mobile-dev-kit-quick-start/cp-mobile-dev-kit-quick-start.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ This step includes creating the mobile development kit project in the editor.
 
     >In [this tutorial](cp-mobile-dev-kit-ms-setup), server-side configuration for this MDK app were already done.
 
-    Regardless of whether you are creating an online or offline application, this step is needed for app to connect to an OData service. When building an MDK Mobile application, it assumes the OData service created and the destination that points to this service is setup in Mobile Services. For MDK Web application, destination is setup in Cloud Platform cockpit.
+    Regardless of whether you are creating an online or offline application, this step is needed for app to connect to an OData service. When building an MDK Mobile application, it assumes the OData service created and the destination that points to this service is setup in Mobile Services. For MDK Web application, destination is setup in SAP BTP cockpit.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-quick-start/cp-mobile-dev-kit-quick-start.md:66]: ...is set up...

Please correct all occurrences.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Many non-native speakers get confused by the difference between "the setup" and "to set (something) up".

I have commented on one line where this mistake has caught my eye. Please check your whole tutorial and correct it.

**Comment by jitendrakansal** [comment]
@MichaelCzcz thanks for reviewing it, this tutorial has been out for more than 1 year,  wonder no developer complained so far. 
please check the updated changes.

File: tutorials/cp-mobile-bas-setup/cp-mobile-bas-setup.md [hunk 2]
```diff
@@ -96,7 +95,7 @@ Your dev space will be created and the status will change to running.
 
     !![Space list](img_3_6.png)
 
-    > Upon successful setup, you will see a toast message at the bottom right corner of your screen, and the bottom status bar will highlight CF connection details.
+    > Upon successful set up, you will see a toast message at the bottom right corner of your screen, and the bottom status bar will highlight CF connection details.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-bas-setup/cp-mobile-bas-setup.md:98]: Here you are referring to the setup as such and not the task. Please undo this change.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thanks for making the changes. You are almost there. Please undo that one change (see comment).

File: tutorials/cp-mobile-bas-setup/cp-mobile-bas-setup.md (hunk 2 above)
Comment (jitendrakansal) [reply tutorials/cp-mobile-bas-setup/cp-mobile-bas-setup.md:98]: done

Comment (jitendrakansal) [reply tutorials/cp-mobile-bas-setup/cp-mobile-bas-setup.md:98]: oh wait 
image

Comment (jitendrakansal) [reply tutorials/cp-mobile-bas-setup/cp-mobile-bas-setup.md]: finally done

**Comment by jitendrakansal** [comment]
@MichaelCzcz please review the last update.
